src/lib/colors/colors.js

```diff
@@ -19,52 +19,62 @@ import htmlColors from './htmlColors.js'
 import { Log } from '../log.js'
 
 export default {
-  normalize(color = '') {
+  // The defaultColor parameter should be set to null when utilizing this function within the template parser.
+  // Because the template parser evaluates attribute values, even if they are not color codes,
+  // to determine whether they represent color values or not.
+  normalize(color = '', defaultColor = '0xffffffff') {
     color = color.toString()
 
-    if (color.startsWith('0x')) {
-      return color
-    }
+    if (color.length > 0) {
```

Comment:
I think this snippet may need to be added back in. All colors, even those that have been normalized during the build step, will still pass through the normalize function at runtime.

So if we do an early return of the ones starting with '0x', then that will save on the execution.

Reply:
Discussed offline and the early return does happen on line 36 (after verifying it's an actual valid color)